_data/committee/2021-22.yaml

```diff
@@ -115,40 +139,53 @@ people:
       Hello, I'm Ryan and I'm going into my 2nd year studying Computer Science. I will be responsible for all of your sports and e-Sports events this year -- 
       please let me know what you want to see! When I'm not sleeping I'm either speedrunning, participating in hackathons or working on programming projects.
 
-  - name: Victoria Tilley
-    pronouns: She/Her, They/Them
-    discordtag: Sapphyre#5656
-    role: Equality, Diversity, and Inclusivity Officer
-    picture: /assets/committee/2021-22/mini/victoria.jpg
-    start-date: 1/7/2021
-    end-date: 28/1/2022
-    bio: >
-      Heya! I'm Victoria and this year I'll be making sure that everyone feels included and represented at all of our events and in our School. 
-      Outside of this, I enjoy game development, airsoft, snowboarding and playing guitar. Come to me if you need anything!
```

Comment:
I think keeping both is what @AlexJBanks wanted

Reply:
Fixed